### PR TITLE
lmms, SafeEjectGPU: add pages

### PR DESCRIPTION
**Pull request by GingkathFox**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/osx/SafeEjectGPU.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# SafeEjectGPU
+
+> Eject a GPU safely.
+> Visit the man page for more info.
```
Comment (navarroaxel) [inline pages/osx/SafeEjectGPU.md:4]: Could we use this man page?
```suggestion
> More information: <https://www.unix.com/man-page/mojave/8/safeejectgpu>.
```

**Comment by bl-ue** [comment]
This PR seems to be a bit messed up.  `dot_clean` was already added in #5144.  Let me see if I can fix it.

**Comment by bl-ue** [comment]
It's cleaned up now.

I see the issue.  @GingkathFox you opened a PR on your fork (GingkathFox#1) and something (I don't know what) got messed up.  To prevent this in the future, run the following commands to update `master`, instead of opening a PR on your fork:
```shell
# only run this one if you don't have the upstream remote
git remote add upstream git@github.com:tldr-pages/tldr.git

# update master
git fetch upstream master
git rebase upstream/master
git push origin
```

File: pages/common/lmms.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`lmms render {{path/to/mmpz_or_mmp/file}}`
+
+- Render the individual tracks of a project file:
+
+`lmms rendertracks {{path/to/mmpz_or_mmp/file}} {{path/to/dump/directory}}`
+
+- Render with custom samplerate, format, and as a loop:
+
+`lmms render --samplerate 88200 --format ogg --loop --output {{path/to/output/file.ogg}}`
```
Comment (bl-ue) [inline pages/common/lmms.md:36]: Missing token syntax?
```suggestion
`lmms render --samplerate {{88200}} --format {{ogg}} --loop --output {{path/to/output/file.ogg}}`
```

File: pages/osx/SafeEjectGPU.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Eject all GPUs:
+
+`SafeEjectGPU Eject`
+
+- List all GPUs attached:
+
+`SafeEjectGPU gpus`
+
+- List apps using a specified GPU:
```
Comment (bl-ue) [inline pages/osx/SafeEjectGPU.md:14]: I think "a" and "specified" together sound a little redundant.
```suggestion
- List apps using a GPU:
```

File: pages/osx/SafeEjectGPU.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`SafeEjectGPU gpuid {{GPU_ID}} apps`
+
+- Get the status of a specified GPU:
```
Comment (bl-ue) [inline pages/osx/SafeEjectGPU.md:18]: Same here.
```suggestion
- Get the status of a GPU:
```

File: pages/osx/SafeEjectGPU.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`SafeEjectGPU gpuid {{GPU_ID}} status`
+
+- Eject a specified GPU:
```
Comment (bl-ue) [inline pages/osx/SafeEjectGPU.md:22]:
```suggestion
- Eject a GPU:
```

File: pages/osx/SafeEjectGPU.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`SafeEjectGPU gpuid {{GPU_ID}} Eject`
+
+- Launch an app on a specified GPU:
```
Comment (bl-ue) [inline pages/osx/SafeEjectGPU.md:26]:
```suggestion
- Launch an app on a GPU:
```

File: pages/common/lmms.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+# lmms
+
+> Render a `.mmp` or `.mmpz` project file, dump a `.mmpz` as XML, or start the GUI.
```
Comment (bl-ue) [inline pages/common/lmms.md:3]: I think this description is a little low level.  A new user likely doesn't know what a `.mmp` or `.mmpz` file is (I didn't :smile:)
```suggestion
> Free, open source, multiplatform digital audio workstation.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hi @GingkathFox!  I've left some comments below for you to review.

File: pages/common/lmms.md (hunk 7 above)
Comment (GingkathFox) [reply pages/common/lmms.md:3]: ah, i was going under the assumption that LMMS users would already know the file formats, but you have a point :D

**Comment by GingkathFox** [comment]
@bl-ue is this change good?

File: pages/common/lmms.md (hunk 7 above)
Comment (bl-ue) [reply pages/common/lmms.md:4]: Of course, but _probably_ most users will see this page by running `tldr lmms` _because_ they have no idea what it is.

Comment (bl-ue) [reply pages/common/lmms.md:4]: That's the way we think when we write page titles, usually.

**Comment by bl-ue** [comment]
@GingkathFox some of the suggestions aren't applied yet.  Please apply those first, then I'll review again :)

**Comment by GingkathFox** [comment]
There we go, done :D

**Review by bl-ue — APPROVED** [review]

**Comment by GingkathFox** [comment]
oh cool, thanks for another manpage site, ill be sure to check that also in future commits

**Review by schneiderl — APPROVED** [review]

**Comment by bl-ue** [comment]
Sorry, this PR got messef up again, and I fixed it by reverting bc66ecc54d1a897f702b858b6077b94585ce0269.  🤷🏻 🤷🏻‍♂️ 🤷🏻‍♀️

**Review by sbrl — COMMENTED** [review]
Note that we've recently made a decision that all filenames should be lowercase if I recall correctly. Could someone open a PR to correct that here please?

**Comment by bl-ue** [comment]
@sbrl Funny! I forgot about that :smile:

@GingkathFox will you please do the honors here? 🙏🏻

**Pull request merged**